engines/cine/gfx.cpp

```diff
@@ -93,6 +93,377 @@ static const byte mouseCursorCross[] = {
+	// English, final credits screen
+	"Delphine team hopes you had a great time with %s and we thank you for your support.\n"
+	"See you soon in our next adventure.",
+
+	// French
+	"Sc\202nario\n Paul Cuisset, Philippe Chastel\n"
+	"Programmation\n Paul Cuisset, Philippe Chastel, J\202sus Martinez, Daniel Morais IBM PC\n"
+	"Graphismes\n Mich\212le Bacqu\202, Imagex, Emmanuel Lecoz\n"
+	"Musique\n Jean Baudlot\n"
+	"Bruitages\n Marc Minier\n"
+	"Documentation notice\n Michael Sportouch\n"
+	"Illustration notice\n Jean-Pierre Fert\202\n"
+	"Packaging\n Philippe Delamarre\n"
+	"Remerciements\n Patricia Vermander, Claire Le Hello, Muriel Fraboni\n",
+	// French, final credits screen
+	"Toute l'\202quipe de Delphine esp\212re que John Glames vous a fait passer de bons moments, "
+	"et vous remercie de votre confiance.\n"
+	"A bient\223t pour une prochaine aventure.",
+
+	// German
+	"Skript\n Paul Cuisset, Philippe Chastel\n"
+	"Programmierung\n Paul Cuisset, Philippe Chastel, J\202sus Martinez, Daniel Morais IBM PC\n"
+	"Illustrationen\n Mich\212le Bacqu\202, Imagex, Emmanuel Lecoz\n"
+	"Musik\n Jean Baudlot\n"
+	"Soundeffekte\n Marc Minier\n"
+	"Bedienungsanleitung\n Michael Sportouch\n"
+	"Illustrationshandbuch\n Jean-Pierre Fert\202\n"
+	"Verpackung\n Philippe Delamarre\n"
+	"Unser Dank gilt\n Patricia Vermander, Claire Le Hello, Muriel Fraboni\n",
+	// German, final credits screen
+	"Delphine Team hofft, Sie hatten eine tolle Zeit mit John Glames und dankt Ihnen f\201r Ihre Unterst\201tzung.\n"
+	"Bis bald bei unserem n\204chsten Abenteuer.",
+
+	// Spanish
+	"Guion\n Paul Cuisset, Philippe Chastel\n"
+	"Programaci\242n\n Paul Cuisset, Philippe Chastel, J\202sus Martinez, Daniel Morais IBM PC\n"
+	"Ilustraciones\n Mich\212le Bacqu\202, Imagex, Emmanuel Lecoz\n"
+	"M\243sica\n Jean Baudlot\n"
+	"Efectos de sonido\n Marc Minier\n"
+	"Manual de usuario\n Michael Sportouch\n"
+	"Manual de ilustraciones\n Jean-Pierre Fert\202\n"
+	"Embalaje\n Philippe Delamarre\n"
+	"Gracias a\n Patricia Vermander, Claire Le Hello, Muriel Fraboni\n",
+	// Spanish, final credits screen
+	"Todo el equipo de Delphine espera que John Glames te haga pasar muy buenos ratos y agradecemos tu confianza.\n"
+	"Hasta la pr\242xima aventura.",
+
+	// Italian
+	"Copione\n Paul Cuisset, Philippe Chastel\n"
+	"Programmazione\n Paul Cuisset, Philippe Chastel, J\202sus Martinez, Daniel Morais IBM PC\n"
+	"Illustrazioni\n Mich\212le Bacqu\202, Imagex, Emmanuel Lecoz\n"
+	"Musica\n Jean Baudlot\n"
+	"Effetti sonori\n Marc Minier\n"
+	"Manuale dell'utente\n Michael Sportouch\n"
+	"Manuale di illustrazione\n Jean-Pierre Fert\202\n"
+	"Confezione\n Philippe Delamarre\n"
+	"Grazie a\n Patricia Vermander, Claire Le Hello, Muriel Fraboni\n",
+	// Italian, final credits screen
+	"Tutto il gruppo della Delphine spera che John Glames vi abbia fatto trascorrere dei momenti piacevoli "
+	"e vi ringrazia della fiducia accordata.\n"
+	"Appuntamento a presto per una nuova aventura."
+};
```

Comment:
I am thinking those arrays could be in a `#ifdef USE_TTS` block so that they are not included when compiling without TTS support (which may include platforms with restricted amount of memory. That would also mean adding the `#ifdef USE_TTS` around the blocks of code below in the same file that handles TTS.

Reply:
Yes, good idea